server/models/preference.go

```diff
@@ -68,7 +68,7 @@ type Preference struct {
 	AnonymousPerfResults      bool                   `json:"anonymousPerfResults"`
 	UpdatedAt                 time.Time              `json:"updated_at,omitempty"`
 	UsersExtensionPreferences map[string]interface{} `json:"usersExtensionPreferences,omitempty"`
-	RemoteProviderPrefs       map[string]interface{} `json:"mesheryCloudPrefs,omitempty"`
+	RemoteProviderPreferences      map[string]interface{} `json:"remoteProviderPreferences,omitempty"`
```

Comment:
@MUzairS15 I updated it to remoteProviderPreferences considering a different remoteProvider might not have Meshery Cloud as its name

Reply:
Is remote provider updated with thks change?

Reply:
yes